### PR TITLE
httpx and aiohttp examples on the front page, close #31

### LINKED ISSUE #31: Add httpx example
There's an `aiohttp` example in quickstart, I think it would be nice to have `httpx` example too.

After all, it's not like one transport is favoured over another, is it?

### PR DESCRIPTION
**Pull request by dimaqq**
* `httpx` and `aiohttp` quick start samples on the front page 
* slightly more prominent `2x faster`

Close #31

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,9 +2,66 @@
 
 [![CircleCI](https://circleci.com/gh/HENNGE/aiodynamo.svg?style=svg)](https://circleci.com/gh/HENNGE/aiodynamo) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
+Asynchronous, pythonic DynamoDB client; **2x** faster than `aiobotocore`.
```
Comment (ojii) [inline readme.md:5]: it's also 2x faster than boto3/botocore fwiw.

File: readme.md [hunk 2]
```diff
@@ -2,9 +2,66 @@
 
 [![CircleCI](https://circleci.com/gh/HENNGE/aiodynamo.svg?style=svg)](https://circleci.com/gh/HENNGE/aiodynamo) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```
Comment (ojii) [inline readme.md:3]: now that the docs link is so much further away, probably should add a docs badge here:

```md
[![Documentation Status](https://readthedocs.org/projects/aiodynamo/badge/?version=latest)](https://aiodynamo.readthedocs.io/en/latest/?badge=latest)
```

File: readme.md (hunk 1 above)
Comment (dimaqq) [reply readme.md]: Hmm sounds like we need to test `aiobotocore` to be sure 🤔 
I'd assume they'd be equally slow or fast, but who knows?

File: readme.md (hunk 2 above)
Comment (dimaqq) [reply readme.md]: Done, thanks!

File: readme.md (hunk 1 above)
Comment (ojii) [reply readme.md]: I ran the benchmarks against all three and the results were comparable. https://github.com/HENNGE/aiodynamo/pull/17#issuecomment-596409057

Comment (ojii) [reply readme.md]: tl;dr: boto3: 7.5s, botocore: 6.6s, aiobotocore: 6.6s, aiodynamo: 3.6s

Comment (dimaqq) [reply readme.md]: Confirmed, scanning 63K rows:
* `aiobotocore`: 19s, 23s, 20s
* `aiodynamo`: 9s, 9s

**Pull request merged**